README.md

```diff
@@ -82,10 +82,13 @@ Note: leading/trailing whitespace will still validate, they will just be trimmed
 
 #### Repository names
 
-- Max length: 100 characters
-- All characters must be either a hyphen (`-`), a period (`.`), or alphanumeric
+- Max length: 100 codepoints
```

Comment:
Could you split this into two words? Unicode uses two words for it, and I'd like to stay consistent with their usage.
```suggestion
- Max length: 100 code points
```